portality/view/doaj.py

```diff
@@ -253,20 +253,27 @@ def toc(identifier=None, volume=None, issue=None):
     # identifier may be the journal id or an issn
     journal = None
     issn_ref = False
+
     if len(identifier) == 9:
```

Comment:
We have the opportunity here to fix another bug - `None` (the default value for `identifier`) will cause an exception if you give it to `len()`. This means there's an error when you go to https://doaj.org/toc/